cysync/src/pages/mainApp/sidebar/portfolio/coins/OneCoin.tsx

```diff
@@ -89,10 +98,18 @@ const OneCoin: React.FC<OneCoinProps> = props => {
   return (
     <Root onClick={onClick} container>
       <Grid item xs={3} className={classes.coin}>
-        <CoinIcons
-          initial={coinInitial.toUpperCase()}
-          style={{ marginRight: '10px' }}
-        />
+        {coinParent ? (
+          <CoinIcons
+            initial={coinInitial.toUpperCase()}
+            parentCoin={coinParent.toLowerCase()}
+            style={{ marginRight: '10px' }}
+          />
+        ) : (
+          <CoinIcons
+            initial={coinInitial.toUpperCase()}
+            style={{ marginRight: '10px' }}
+          />
+        )}
```

Comment:
This can be simplified into something like

```
          <CoinIcons
              initial={coinInitial.toUpperCase()}
              parentCoin={coinParent?.toLowerCase()}
              style={{ marginRight: '10px' }}
            />
 
```